packages/playwright-crawler/src/internals/utils/playwright-utils.ts

```diff
@@ -308,4 +444,5 @@ export const playwrightUtils = {
     gotoExtended,
     blockRequests,
     parseWithCheerio,
+    infiniteScroll,
```

Comment:
you also need to add it to the `PlaywrightContextUtils` interface (so it gets added to the crawling context type automatically) plus register it there in the `registerUtilsToContext` function, both just above this variable